sr_port/cmidef.h

```diff
@@ -1,6 +1,10 @@
 /****************************************************************
  *								*
- *	Copyright 2001, 2004 Sanchez Computer Associates, Inc.	*
+ * Copyright (c) 2001-2004 Fidelity National Information	*
```

Comment:
Same question. Is it okay for us to change a Sanchez copyright?

Reply:
Fixed